### ISSUE TITLE
md-autocomplete items list disappears when clicking on scrollbar

### ISSUE THREAD

**Issue by webmandman** [angular/material#1739]
md-autocomplete items list disappears when I click on scrollbar. The demo is doing exactly the same thing.

![issie-md-autocomplete-listscrollbar](https://cloud.githubusercontent.com/assets/991677/6434515/9202f166-c03f-11e4-9404-e85da611d0b5.gif)

**Comment by rosscdh** [comment]
+1

**Comment by rosscdh** [comment]
seems to happen on items with me too (sorta)

![auto-complete-weirdness](https://cloud.githubusercontent.com/assets/397106/6442764/a084099c-c0f2-11e4-82f9-9b274396fb0c.gif)

**Comment by ThomasBurleson** [comment]
@robertmesserle - do you think this should be fixed for 0.8.3 ?

**Comment by rosscdh** [comment]
Would be much appreciated.

Also some way to acces the autocomplete controller methods? On-select hide
behaviour is missing.

**Comment by webmandman** [comment]
Thank you Robert!

**Comment by webmandman** [comment]
Robert, tried your fix and it worked. 

Now I found that when I want to skipped on selecting an item and instead get out of the item list I cannot. 
![issue-md-autocomplete-list-doesnot-hide](https://cloud.githubusercontent.com/assets/991677/6452937/f02c02a6-c0ed-11e4-9653-3087152a34b3.gif)

**Comment by robertmesserle** [comment]
@webmandman Good catch, added a fix for this case as well.  Please let me know if anything else shows up.

**Comment by webmandman** [comment]
This was fixed back in 0.8.3, but now that I've updated to 0.11.0 its no longer working. The exact same thing is happening. Clicking on the scrollbars hides the autocomplete dropdown.

**Comment by webmandman** [comment]
Oh, its working fine on Chrome, but not for Internet Explorer 11(edge). 
![md-autocomplete-ie-issue](https://cloud.githubusercontent.com/assets/991677/10002521/7749ef10-605c-11e5-87ae-78ede3f34990.gif)

**Comment by webmandman** [comment]
Oh, it works fine in Chrome, but not in IE 11(edge).
![md-autocomplete-ie-issue](https://cloud.githubusercontent.com/assets/991677/10002704/8b84d93a-605d-11e5-90f9-db92d18a7b53.gif)

**Comment by webmandman** [comment]
@robertmesserle I know this was fixed for 0.8.3, but yesterday I updated to 0.11.0 and it stopped working for IE 11 edge, still works fine in Chrome.

Still the same issue as MattgyverHub, but this can't be about the fix not being commited or applied. The autocompleteController.js file has been updated over 40 times and in 0.11.0 it still has the same logic. Some where along it stopped working for IE. 

Do I create a new issue?
